tests/framework/Optimizers/GeneticAlgorithms/continuous/unconstrained/ZDT1-static-1.0Xover-0.0Mutation.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug" profile="jobs">

```

Comment:
Another suggestion for all the tests, you may want to leverage external xml to avoid duplicates of tests with minor changes.